src/amber/router/context.cr

```diff
@@ -122,6 +122,8 @@ class HTTP::Server::Context
   end
 
   protected def finalize_response
+    response.headers["Connection"] = "Keep-Alive"
```

Comment:
@eliasjpr Currently crystal sends back `Connection: Keep-Alive` by default regardless of request header which usually doesn't actually contain a connection header as it is a Server thing that tells the browser what it's ok with. I'm not sure why but changing it to `Connection: Close` is a huge performance hit. Slower than rails....

Changing the timeout reduces the number of File descriptors dramatically while still being fast. I believe that this is the best course of action since going from 115,000 rps to 1500 is not ok in my book.